webapp/security/models.py

```diff
@@ -37,11 +61,59 @@
     Column("release_id", Integer, ForeignKey("release.id")),
 )
 
+cve_bugs = Table(
+    "cve_bugs",
+    Base.metadata,
+    Column("cve_id", String, ForeignKey("cve.id")),
+    Column("bug_id", Integer, ForeignKey("bug.id")),
+)
+
+cve_references = Table(
+    "cve_references",
+    Base.metadata,
+    Column("cve_id", String, ForeignKey("cve.id")),
+    Column("cve_reference_id", Integer, ForeignKey("cve_reference.id")),
+)
+
+cve_packages = Table(
+    "package_release_status",
+    Base.metadata,
+    Column("package_id", Integer, ForeignKey("package.id")),
+    Column("name", Integer, ForeignKey("release.id")),
+    Column("type", Enum(PackageType)),
+)
+
+package_release_status = Table(
+    "package_release_status",
+    Base.metadata,
+    Column("package_id", Integer, ForeignKey("package.id")),
+    Column("release_id", Integer, ForeignKey("release.id")),
+    Column("status", Enum(PackageStatus)),
+    extend_existing=True,
+)
+
 
 class CVE(Base):
     __tablename__ = "cve"
 
-    id = Column(String, primary_key=True)
+    id = Column(String, primary_key=True, unique=True)
```

Comment:
I believe `unique=True` is not necessary here case the `primary_key=True` constraint already enforces uniqueness.

Reply:
ok, didn't know that